src/scripts.js

```diff
@@ -98,30 +98,30 @@ fetchData('users/userData')
           .getContext('2d'), {
           type: 'line',
           data: {
-            labels: dropYear(sleep.returnWeekDatesOnly(1)),
+            labels: dropYear(sleep.returnWeekDatesOnly(today)),
```

Comment:
So happy with this change. The variable rather than a number is so much more readable!